Mathlib/Init/Data/Rat/Basic.lean

```diff
@@ -11,7 +11,5 @@ import Mathlib.Mathport.Rename
 # Align statements for declarations from Std
 -/
 
-#align rat.ext Rat.ext
 #align rat Rat
-#align rat.ext_iff Rat.ext_iff
```

Comment:
Why is this needed?

Reply:
The `@[ext]` attribute was removed in Std, so these don't exist. We should change to #noalign, however.

Reply:
Hmm, but that can't be right, because `Rat.ext` exists later.

Reply:
Okay, sorted. Rat.ext still exists, in a differnet file in Std, ext_iff has been removed.

Reply:
Surely we should keep `ext_iff`?